client/main.js

```diff
@@ -3,14 +3,15 @@ import App from './App.vue';
 import store from './store';
 import router from './router';
 import VueResource from 'vue-resource';
+import VueFlatpickr from 'vue-flatpickr';
+import 'vue-flatpickr/theme/dark.css';
 
 Vue.use(VueResource);
+Vue.use(VueFlatpickr);
   
 new Vue({
   el: '#app',
   store,
   router,
-  components: {
-    App
-  }
+  render: h => h(App)
```

Comment:
Using the render function instead of registering component I don't have to have an `app` tag in my html which was more attractive to me so vue-template code is contained only inside .vue components if all that makes sense